pages/requests/create-renewal.tsx

```diff
@@ -0,0 +1,208 @@
+import Layout from '@components/admin/Layout'; // Layout component
+import { Text, Box, Flex, Stack, Button, GridItem, Input } from '@chakra-ui/react'; // Chakra UI
+import { useState } from 'react'; // React
+import PermitHolderInformationForm from '@components/admin/requests/forms/PermitHolderInformationForm'; //Permit holder information form
+import { PermitHolderInformation } from '@tools/components/admin/requests/forms/types'; //Permit holder information type
+import DoctorInformationForm from '@components/admin/requests/forms/DoctorInformationForm'; //Doctor information form
+import { DoctorInformation } from '@tools/components/admin/requests/forms/doctor-information-form'; //Doctor information type
+import AdditionalQuestionsForm from '@components/admin/requests/forms/renewals/AdditionalQuestionsForm'; //Additional questions form
+import { AdditionalQuestions } from '@tools/components/admin/requests/forms/types'; //Additional questions type
+import PaymentDetailsForm from '@components/admin/requests/forms/PaymentDetailsForm'; //Payment details form
+import { PaymentDetails } from '@tools/components/admin/requests/forms/types'; //Payment details type
+import { PaymentType, Province } from '@lib/graphql/types'; //GraphQL types
+
+export default function CreateRenewal() {
+  const [permitHolderID] = useState('303240');
```

Comment:
```suggestion
  const [permitHolderID] = useState(303240);
```
Make this a number type